src/main/java/com/example/demo/GlobalExceptionHandler.java

```diff
@@ -64,4 +65,12 @@ public ResponseEntity<ErrorResponseBody> handleUserNotCreatedException(UserNotCr
         return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(errorResponseBody);
     }
 
+        @ExceptionHandler(PostNotDeletedException.class)
+    public ResponseEntity<ErrorResponseBody> handlPostNotDeletedException(PostNotDeletedException e) {
```

Comment:
`handlPostNotDeletedException` -> `handlePostNotDeletedException`

おそらくtypoだと思います。IDE上で黄色い波線などが出ているかと思いますので、気にかけるようにしてください。

```suggestion
    public ResponseEntity<ErrorResponseBody> handlePostNotDeletedException(PostNotDeletedException e) {
```